cdk/src/open-data-platform/frontend/frontend-stack.ts

```diff
@@ -0,0 +1,49 @@
+// Defines the s3 Bucket to host static assets, the distribution which
+// points to that bucket, and deploys Vue app to the bucket.
+//
+// Based on https://github.com/BlueConduit/patina/blob/main/cdk/lib/patina-stack.ts,
+// which gets bucket and distribution references from
+// https://github.com/BlueConduit/tributary/blob/7a275259c484dd637467c841a62997cb6370c9f4/cdk/lib/frontend/frontend-stack.ts.
+
+
+import {Stack} from 'aws-cdk-lib';
+import {Construct} from "constructs";
+import * as s3 from "aws-cdk-lib/aws-s3";
+import * as cloudfront from "aws-cdk-lib/aws-cloudfront";
+import * as origins from "aws-cdk-lib/aws-cloudfront-origins";
+import * as s3deploy from "aws-cdk-lib/aws-s3-deployment";
+import {CommonProps} from '../../util';
+
+export class FrontendStack extends Stack {
+
+  readonly distribution: cloudfront.Distribution;
+  readonly vueAssetsBucket: s3.Bucket;
+
+  constructor(scope: Construct, id: string, props: CommonProps) {
+    super(scope, id, props);
+
+    // Create s3 bucket to host static assets.
+    this.vueAssetsBucket = new s3.Bucket(this, "VueAssets");
```

Comment:
Could you remove the reference to `vue` here, so it's more generic? e.g. `FrontendAssets`. Unless there's something that is Vue-specific about the bucket.